### DISCUSSION TITLE
Hilight on a system message?

### DISCUSSION THREAD

**Discussion by TheRandomRock** [Chatterino/chatterino2#4859] [Q&A] [answered]
Any way to get a system message in the /mention tab? The patterns entered for highlights does not pick up system messages.

**Comment by Felanbird** [comment, marked as answer]
No this isn't possible, you'd have to use filters on a second instance of the chat

![image](https://github.com/Chatterino/chatterino2/assets/41973452/a5f178f4-4fea-4df9-bf5f-917d8e858e93)

`flags.system_message && message.content contains "timed"` I'm using this filter in the example to show only timeouts in that tab.
